src/ngx_http_lua_sslcertby.c

```diff
@@ -719,6 +719,38 @@ ngx_http_lua_ffi_ssl_server_name(ngx_http_request_t *r, char **name,
     return NGX_DECLINED;
 }
 
+int
+ngx_http_lua_ffi_privkey_pem_to_der(const u_char *pem, size_t pem_len, u_char *der,
```

Comment:
Style: this line exceed 80 columns. Let's break it up properly.

BTW, I usually use the [ngx-releng](https://github.com/openresty/nginx-devel-utils/blob/master/ngx-releng) tool to detect such things.